src/components/ChecImageManager.vue

```diff
@@ -52,16 +56,52 @@ export default {
   },
   mixins: [dropzone],
   props: {
+    /**
+     * Foot note to be displayed under the button. eg: PNG, JPG, & GIFS accepted.
+     */
     footnote: {
       type: String,
       default: null,
     },
+    /**
+     * Number of columns to be displayed. either 2 or 4.
+     */
+    columns: {
+      type: Number,
+      default: 4,
+      validator(value) {
+        return [2, 4].includes(value);
```

Comment:
Did you need this validator? Seems like an unnecessary restriction...

Reply:
No, I suppose not, I've removed it.